### PR TITLE
TF Keras vis_utils_test introducing test cases

### PR DESCRIPTION
**Pull request by Dayananda-V**
Test case introducing for vis_utils.py file under keras package.

### CODE REVIEW

**Comment by Dayananda-V** [comment]
@tanzhenyu 

Please help to review the test case

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @tanzhenyu: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

File: tensorflow/python/keras/utils/vis_utils_test.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+from __future__ import print_function
+
+from tensorflow.python import keras
+from tensorflow.python.keras.utils import vis_utils
+from tensorflow.python.lib.io import file_io
+from tensorflow.python.platform import test
+
+class ModelToDotFormatTest(test.TestCase):
+
+  def test_plot_model(self):
```
Comment (tanzhenyu) [inline tensorflow/python/keras/utils/vis_utils_test.py]: Can you split this into two different unit tests? Otherwise LGTM to me.

Comment (Dayananda-V) [reply tensorflow/python/keras/utils/vis_utils_test.py]: same functional test so kept under single unit test case. Hope this not required to split in to two different unit test.

**Review by tanzhenyu — DISMISSED** [review]

**Comment by rthadur** [comment]
@Dayananda-V please check failed tests

**Comment by tanzhenyu** [comment]
This is been failing unit test -- can you investigate?

**Comment by Dayananda-V** [comment]
@tanzhenyu 

fixed compilation problem and please review once again, TIA!......

**Review by tanzhenyu — APPROVED** [review]
